assets/js/script.js

```diff
@@ -0,0 +1,62 @@
+
+const bodySelector =  document.querySelector('body');
+const gameSection = document.querySelector('#game-section')
+
```

Comment:
I noticed the start button seems missing
```suggestion
const startBtn = document.querySelector('#start-button');
```

Reply:
oh wow, i deleted it by accident, thank you Bryan!